GVFS/GVFS.Common/Git/GitProcess.cs

```diff
@@ -760,6 +755,11 @@ public Process GetGitProcess(string command, string workingDirectory, string dot
             }
         }
 
+        private static string GenerateCredentialVerbCommand(string verb)
+        {
+            return $"-c {GitConfigSetting.CredentialUseHttpPath}=true credential {verb}";

```

Comment:
Why are we not just setting this in the git config for the repo?  It shouldn't affect other git commands right? And should only be used for `dev.azure.com`?  Maybe we should check the repo url on clone and mount and set it in the config?  How will this affect clones done against GitHub?

Reply:
Thanks Kevin -

> Why are we not just setting this in the git config for the repo?

1) It is set on mount
2) Not all config commands that deal with credentials are run in the context of a repository.

> And should only be used for dev.azure.com

It is scoped to dev.azure.com, so will only affect dev.azure.com urls. Maybe tweaking the variable name would make this more clear? There could be a discussion about whether we should even set this if we know the URL is not a dev.azure.com URL, but:

1) It doesn't hurt to always set this (git will take it and only apply it to the expected urls)
2) It is arguably more consistent to always apply this in VFS4G
3) It is closer to where we want to end eventually

Reply:
Okay thanks Jameson.  Can we just make add this to the const and not have this helper method like so
`public const string CredentialUseHttpPath = "-c credential.\"https://dev.azure.com\".useHttpPath=true";`
And then use a
`$"{CredentialUseHttpPath} credential fill"` etc in the invocations of git?

Reply:
I like this suggestion, but we also use the the setting name directly in `TrySetRequiredGitConfigSettings`:

```
{ GitConfigSetting.CredentialUseHttpPath, "true" },
```

Reply:
Then how about another `const` for `CommandLineDevAzureCredentialUseHttpPath` that is
`$"-c {CredentialUseHttpPath}=true";` or similar?